keyboards/3w6/keymaps/helltm/keymap.c

```diff
@@ -0,0 +1,170 @@
+    * | CPS |-----'     '-----|_____|                         |_____|-----'     '-----| Del |
+    * '-----'          .-----. _____   .-.._           _..-.   _____ .-----.          '-----'
+    *  SHIFT           |     ||CTRL | /     /         \     \ | BSp ||     |           SHIFT
+    *                  '-----':_____|/ ENT /           \ SPC \|_____:'-----'
+    *                               '--.._/             \_..--'
+    */
+    [_NAV] = LAYOUT(
+        KC_ESC         , KC_VOLD, KC_MUTE, KC_VOLU, XXXXXXX      ,                                XXXXXXX, KC_PGDN, KC_UP  , KC_PGUP, KC_BSPC,
+        KC_TAB         , KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX      ,                                KC_HOME, KC_LEFT, KC_DOWN, KC_RGHT, KC_QUOT,
+        LSFT_T(KC_CAPS), XXXXXXX, KC_MSTP, XXXXXXX, KC_THUMB_SWAP,                                XXXXXXX, XXXXXXX, XXXXXXX, KC_END , RSFT_T(KC_DEL) ,
+                                              XXXXXXX, _______, _______,                    _______, KC_BSPC, XXXXXXX
+    ),
+
+    /*
+    * Numbers
+    *              _____                                                   _____
+    *       .-----|  3  |-----._____                           _____.-----|  8  |-----.
+    * .-----|  2  |_____|  4  |  5  |                         |  6  |  7  |_____|  9  |-----.
+    * |  1  |-----| F3  |-----|_____|                         |_____|-----| F8  |-----|  0  |
+    * |-----| F2  |_____| F4  | F5  |                         | F6  | F7  |_____| F9  |-----|
+    * | F1  |-----| Scr |-----|_____|                         |_____|-----|     |-----| F10 |
+    * |-----| TMg |_____|     |     |                         |     |     |_____| F11 |-----|
+    * |Shift|-----'     '-----|_____|                         |_____|-----'     '-----| F12 |
+    * '-----'          .-----. _____   .-.._           _..-.   _____ .-----.          '-----'
+    *                  |     ||CTRL | /     /         \     \ | BSp ||     |           SHIFT
+    *                  '-----':_____|/ ENT /           \ SPC \|_____:'-----'
+    *                               '--.._/             \_..--'
+    */
+    [_NUM] = LAYOUT(
+        KC_1   , KC_2       , KC_3      , KC_4   , KC_5   ,                                KC_6   , KC_7   , KC_8   , KC_9   , KC_0   ,
+        KC_F1  , KC_F2      , KC_F3     , KC_F4  , KC_F5  ,                                KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 ,
+        KC_LSFT, LCA(KC_DEL), SWIN(KC_S), XXXXXXX, XXXXXXX,                                XXXXXXX, XXXXXXX, XXXXXXX, KC_F11 , RSFT_T(KC_F12) ,
+                                       XXXXXXX, _______, _______,                    _______, KC_BSPC, XXXXXXX
+    ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+
+        /* Space Enter swap */
+        case KC_THUMB_SWAP:
+            if (record->event.pressed) {
+                swap_space_enter = !swap_space_enter;
```

Comment:
It may be worth having it call clear_keyboard, or otherwise unregistering space and enter, here.  To make sure you don't end up with stuck keys.

Reply:
used the very same code on the lime for a bit now and didn't have any issues but, just to be sure, I'll take a look at it tomorrow